components/clp-package-utils/clp_package_utils/scripts/native/compress.py

```diff
@@ -205,9 +208,10 @@ def main(argv):
     )
     if parsed_args.remove_path_prefix:
         clp_input_config.path_prefix_to_remove = parsed_args.remove_path_prefix
-    clp_io_config = ClpIoConfig(
-        input=clp_input_config, output=OutputConfig.parse_obj(clp_config.archive_output)
-    )
+    clp_output_config = OutputConfig.parse_obj(clp_config.archive_output)
+    if parsed_args.tags:
+        clp_output_config.tags = parsed_args.tags.split(",")
```

Comment:
What about just ignoring empty tags?

Reply:
Sure. Maybe we should validate that if `--tags` is specified, there's at least one (non-empty) tag? E.g., if the user uses `--tags $var` but they have a typo if their shell variable, then it would be nice for them to get an error that they didn't specify a tag rather than we create untagged archives.

Reply:
If there's a typo, `--tags $var` becomes `--tags` and the parser will report an error.

Reply:
They could quote it, right? `--tags "$var"` which becomes `--tags ""`.

Reply:
Right, but that also applies to other options (e.g. `timestamp-key`), do we want to also check them?

Reply:
Yeah, we should be, lol.

Reply:
What about submitting a small PR to check all of them?

Reply:
Sounds good